lib/index.js

```diff
@@ -14,10 +14,15 @@ module.exports.__tests = {
 function inspect(root, targetFile, options) {
   if (!options) { options = {}; }
   var command = options.command || 'python';
+  var baseargs = [];
+  if (targetFile === 'Pipfile') {
+    command = 'pipenv';
```

Comment:
it's cool ... but we need to think what if the `pipenv` command is not available. At minimum print a nice error, or maybe call it only if it exists - otherwise just call `pip_resolve.py` directly?

Reply:
I'll print a nice error.

I don't think calling `python pip_resolve.py` would be the right thing to do in this case; it could use the wrong environment and give confusing results.